omod/src/main/resources/config.xml

```diff
@@ -90,6 +90,11 @@
 		<defaultValue>false</defaultValue>
 		<description>Scheduler marks scheduled appointments as missed when it is turned on</description>
 	</globalProperty>
+	<globalProperty>
+	<property>SchedulerReminderBeforeHours</property>
+	<defaultValue>24</defaultValue>
+	<description>Before how many hours the reminder is to be scheduled for appointment</description>

```

Comment:
Rename this description to: `How many hours before the appointment should the reminder SMS be sent`